src/api/app/assets/javascripts/webui2/ignored_requests.js

```diff
@@ -0,0 +1,5 @@
+function setIgnoredRequestDialog() {
```

Comment:
that doesn't really sound like deserving its own function + file, but fine.

Reply:
> that doesn't really sound like deserving its own function + file, but fine.

No problem. File removed, and code kept in the same .haml file.